helm/values.yaml

```diff
@@ -7,10 +7,24 @@ global:
   metrics: {}
   environment: ''
   jobDefinitions: {}
-  serviceUrls:
-    jobManager: ""
-    heartbeatManager: ""
-    polygonPartsManager: ""
+  serviceUrls: {}
+
+jobDefinitions:
+  jobs:
+    new:
+      type: ""
+    update:
+      type: ""
+    swapUpdate:
+      type: ""
+  tasks:
+    polygonParts:
+      type: ""
+
+serviceUrls:
+  jobManager: ""
+  heartbeatManager: ""
+  polygonPartsManager: ""
 
```

Comment:
add tracing:
  enabled: false
  url: ''

metrics:
  enabled: false
  url: ''

Reply:
+ add support for prometheus scrape and port